common/src/main/java/com/wynntils/core/keybinds/KeyBindManager.java

```diff
@@ -38,6 +39,15 @@ public static void onKeyPress(InventoryKeyPressEvent e) {
         });
     }
 
+    @SubscribeEvent
+    public static void onMousePress(InventoryMouseClickedEvent e) {
+        KEY_BINDS.forEach(keyBind -> {
+            if (keyBind.getKeyMapping().key.getValue() == e.getButton()) {
```

Comment:
I think you can use `matchesMouse()` instead. Also means you don't have to expose `key`.